azure-simple/packer/ansible-ei/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Do we need this file since we are adding the License file in the root directory?

Reply:
Removed.